src/main/java/ir/sahab/rest/common/security/SecurityUtil.java

```diff
@@ -0,0 +1,35 @@
+package ir.sahab.rest.common.security;
+
+import java.util.Collection;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.context.SecurityContextHolder;
+
+/**
+ * Utility class with a collection of static methods which help to do security-related things like getting current user
```

Comment:
Change the Javadoc:

A utility class that simplifies getting name or checking authorities of current authenticated principal.